### ISSUE TITLE
New build system - htmllint errors (on Windows only?)

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#22535]
Since updating to the new `npm` based build system, any attempts at building locally (just using `npm test`) on my Windows machine results in a long list of errors thrown by htmllint

```
_gh_pages/components/collapse/index.html: line 1312, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1313, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1314, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1315, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1316, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1317, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1320, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1321, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1322, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1323, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1324, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1325, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1326, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1327, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1328, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1329, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1330, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1331, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1371, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1372, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1373, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1374, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1377, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1378, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1379, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1380, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1381, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1382, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1383, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1384, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1385, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1386, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1387, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1389, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1390, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1391, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1392, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1393, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/collapse/index.html: line 1394, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/card/index.html: line 1117, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1188, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1228, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1625, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1638, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1664, col 3, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1919, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1927, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1935, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1976, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1986, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 1996, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2047, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2055, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2063, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2104, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2114, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2124, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2177, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2194, col 5, a source must be given for each `img` tag
_gh_pages/components/card/index.html: line 2219, col 5, a source must be given for each `img` tag
_gh_pages/components/forms/index.html: line 1684, col 7, label does not have a `for` attribute or a labeable child
_gh_pages/components/forms/index.html: line 2068, col 5, label does not have a `for` attribute or a labeable child
_gh_pages/components/forms/index.html: line 2099, col 5, label does not have a `for` attribute or a labeable child
_gh_pages/components/list-group/index.html: line 1493, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/list-group/index.html: line 1494, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/list-group/index.html: line 1495, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/list-group/index.html: line 1496, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/list-group/index.html: line 1502, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/list-group/index.html: line 1503, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1615, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1616, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1617, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1618, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1619, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1620, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1623, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1624, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1625, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1626, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1627, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1628, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1629, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1630, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1631, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1632, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1633, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1634, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1635, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1636, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1637, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1638, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1639, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1640, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1641, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1642, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1643, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1644, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1645, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1646, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1698, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1699, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1700, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1701, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1704, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1705, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1706, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1707, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1708, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1709, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1710, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1711, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1712, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1713, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1714, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1715, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1716, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1717, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1718, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/modal/index.html: line 1719, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1325, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1326, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1327, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1328, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1329, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1330, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1333, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1334, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1335, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1336, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1337, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1338, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1339, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1340, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1341, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1342, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1343, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1344, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1345, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1346, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1347, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1348, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1349, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1350, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1351, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1352, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1353, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1354, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1355, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1356, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1357, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1358, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1359, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1360, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1361, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1362, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1363, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1420, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1421, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1422, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1423, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1426, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1427, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1428, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1429, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1430, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1431, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1432, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1433, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/carousel/index.html: line 1113, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1116, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1119, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1146, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1149, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1152, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1200, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1203, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1206, col 7, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1265, col 9, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1272, col 9, a source must be given for each `img` tag
_gh_pages/components/carousel/index.html: line 1279, col 9, a source must be given for each `img` tag
_gh_pages/components/navs/index.html: line 1887, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/navs/index.html: line 1888, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/navs/index.html: line 1889, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/navs/index.html: line 1890, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/navs/index.html: line 1896, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/navs/index.html: line 1897, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/popovers/index.html: line 1279, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/popovers/index.html: line 1280, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/popovers/index.html: line 1281, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/popovers/index.html: line 1413, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/popovers/index.html: line 1416, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/tooltips/index.html: line 1244, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/tooltips/index.html: line 1251, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/tooltips/index.html: line 1252, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/tooltips/index.html: line 1253, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1353, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1354, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1355, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1356, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1357, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1358, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1361, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1362, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1363, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1364, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1365, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1366, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1374, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1375, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1376, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1377, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1380, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1381, col 1, indenting spaces must be used in groups of 2
_gh_pages/components/scrollspy/index.html: line 1382, col 1, indenting spaces must be used in groups of 2
_gh_pages/content/figures/index.html: line 1070, col 3, a source must be given for each `img` tag
_gh_pages/content/figures/index.html: line 1083, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1080, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1096, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1106, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1107, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1114, col 3, a source must be given for each `img` tag
_gh_pages/content/images/index.html: line 1121, col 5, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 64, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 68, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 72, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 77, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 81, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 85, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 90, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 94, col 13, a source must be given for each `img` tag
_gh_pages/examples/album/index.html: line 98, col 13, a source must be given for each `img` tag
_gh_pages/examples/carousel/index.html: line 132, col 11, a source must be given for each `img` tag
_gh_pages/examples/carousel/index.html: line 144, col 11, a source must be given for each `img` tag
_gh_pages/examples/carousel/index.html: line 156, col 11, a source must be given for each `img` tag
_gh_pages/examples/jumbotron/index.html: line 75, col 1, indenting spaces must be used in groups of 2
_gh_pages/examples/justified-nav/index.html: line 78, col 1, indenting spaces must be used in groups of 2
_gh_pages/getting-started/introduction/index.html: line 1155, col 1, indenting spaces must be used in groups of 2
_gh_pages/layout/media-object/index.html: line 1083, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1112, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1119, col 9, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1153, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1172, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1191, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1218, col 3, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1236, col 5, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1243, col 5, a source must be given for each `img` tag
_gh_pages/layout/media-object/index.html: line 1250, col 5, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1091, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1092, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1093, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1094, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1095, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1096, col 3, a source must be given for each `img` tag
_gh_pages/utilities/borders/index.html: line 1097, col 3, a source must be given for each `img` tag
_gh_pages/utilities/sizing/index.html: line 1097, col 1, a source must be given for each `img` tag
_gh_pages/utilities/typography/index.html: line 1116, col 5, a source must be given for each `img` tag

[htmllint] found 256 errors out of 102 files
```

but it seems i'm the only one getting these? https://github.com/twbs/bootstrap/pull/22534#issuecomment-297684272

this is on my local branch, which is sync'd and rebased with the latest `v4-dev` here

i submitted https://github.com/twbs/bootstrap/pull/22500 to fix the above linting errors, but now i'm more fundamentally wondering why others may not be getting these linting errors. thoughts? /cc @Johann-S

**Comment by Johann-S** [comment]
You're not alone @patrickhlauke when I ran `npm test` just like you I have same errors and I'm on Windows 10 too

**Comment by patrickhlauke** [comment]
Ah ok...phew. In which case I'm hoping @mdo can have a look at #22500 soon, as at the moment I can't fix things/submit substantive PRs while trying to use the new build system... (unless I run some local modification on my branches, which I'd like to avoid...)

**Comment by peterblazejewicz** [comment]
FYI `on Windows only?` - current branch passes on my OS X `[htmllint] found 0 errors out of 21 files`

**Comment by wolfy1339** [comment]
Current `v4-dev` passes on Ubuntu 16.04.2, so this confirms that this is a Windows only problem

**Comment by patrickhlauke** [comment]
so how can this be a windows only issue? is some tool before the lint step not generating the same output? is the linting not happening the same way? because the errors the linter is reporting are indeed valid errors, and i'm surprised they've not been picked up before...

**Comment by patrickhlauke** [comment]
@peterblazejewicz `[htmllint] found 0 errors out of 21 files` how come it's only linting 21 files? i'm sure the full docs comprise more than just 21 html files...

**Comment by wolfy1339** [comment]
Could it be possible that Jekyll is giving different outputs between Window and other OS's

**Comment by patrickhlauke** [comment]
now that https://github.com/twbs/bootstrap/pull/22500 is merged, i don't get the errors anymore...but intrigued still why it seemed to only happen on windows.

if the errors were just about spaces, i could have understood potential win/macOS/ubuntu differences. but as the linting errors were quite substantive (`<label>` without a `for` attribute etc), and clearly present in the markdown files, i think the issue may be more that the linter on macOS/ubuntu wasn't actually linting the right files/not all of them?

**Comment by peterblazejewicz** [comment]
> @peterblazejewicz [htmllint] found 0 errors out of 21 files how come it's only linting 21 files? i'm sure the full docs comprise more than just 21 html files... [edit: and note in my case, it found errors in 102 files]

@patrickhlauke 
This works better on OS X. Most recent branch version:
```
"docs-lint": "htmlhint --config docs/.htmlhintrc _gh_pages/ js/tests/visual/ && htmllint --rc docs/.htmllintrc --cwd _gh_pages/ && htmllint --rc docs/.htmllintrc --cwd js/tests/visual/",
```
```bash
npm run docs-lint
> bootstrap@4.0.0-alpha.6 docs-lint /Users/piotrblazejewicz/git/bootstrap
> htmlhint --config docs/.htmlhintrc _gh_pages/ js/tests/visual/ && htmllint --rc docs/.htmllintrc --cwd _gh_pages/ && htmllint --rc docs/.htmllintrc --cwd js/tests/visual/
   Config loaded: docs/.htmlhintrc
   Config loaded: docs/.htmlhintrc
Scanned 101 files, no errors found (1626 ms).
[htmllint] found 0 errors out of 91 files
[htmllint] found 0 errors out of 10 files
```

**Comment by bardiharborow** [comment]
I think the issue here is that the HTML linters glob (asterisk wildcards) support is poor and on bash-based systems, bash expands the globs correctly, but on Windows the linter itself expands the glob incorrectly.

**Comment by patrickhlauke** [comment]
> but on Windows the linter itself expands the glob incorrectly.

however, note that the linter on windows DID find actual invalid HTML in the docs output. so i'd argue that the end result is more accurate on windows, so if it's somehow possible to change this part of the build script to behave like it does on windows even on a mac/linux machine, that'd be best (otherwise there'll always be the problem of further errors in the docs output that go undiscovered until somebody on windows runs the build script)

**Comment by XhmikosR** [comment]
This doesn't happen for me.

Can we close it @patrickhlauke @bardiharborow ?

**Comment by patrickhlauke** [comment]
with all the very latest npm stuff, i get a clean `[htmllint] found 0 errors out of 109 files` ... assuming on macOS the situation is similar (and that it's indeed checking all the files, not just `21 files` as in https://github.com/twbs/bootstrap/issues/22535#issuecomment-297859382) then yes, closing this issue as resolved now.

**Comment by XhmikosR** [comment]
@patrickhlauke: the latter I fixed it a couple of days ago, see #24241. Surprisingly on Windows it was fine that is why it took me some time to notice it.
